sdl-core/src/main/scala/io/smartdatalake/workflow/dataobject/JsonFileDataObject.scala

```diff
@@ -67,7 +56,7 @@ case class JsonFileDataObject( override val id: DataObjectId,
                                override val schemaMin: Option[GenericSchema] = None,
                                override val saveMode: SDLSaveMode = SDLSaveMode.Overwrite,
                                override val sparkRepartition: Option[SparkRepartitionDef] = None,
-                               stringify: Boolean = false,
+                               @Deprecated @deprecated("Use action/transformers instead", "2.6.0") stringify: Boolean = false,
```

Comment:
Maybe a stupid question, but is the double deprecated necessary?

Reply:
Uppercase "@Deprecated" is the java annotation, and lowercase is Scala.
Normally we just use the scala variant, but in order to mark the attribute in the schema export / schema viewer as deprecated we need also the Java variant. This is due to the Scala variant being only available at compile time, but not for reflection in the schema export which runs as separate cmd after compilation...